platform/viewer/src/connectedComponents/Viewer.js

```diff
@@ -58,20 +59,22 @@ class Viewer extends Component {
   static propTypes = {
     studies: PropTypes.array,
     studyInstanceUids: PropTypes.array,
+    activeServer: PropTypes.object,
```

Comment:
Can we do a `PropTypes.shape({ })`, with our expected fields spelled out more explicitly?